opensrp-app/src/main/java/org/smartregister/util/SyncUtils.java

```diff
@@ -0,0 +1,86 @@
+package org.smartregister.util;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.ResolveInfo;
+import android.util.Base64;
+import android.util.Log;
+
+import org.apache.http.HttpStatus;
+import org.smartregister.CoreLibrary;
+import org.smartregister.R;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.List;
+
+import static org.smartregister.AllConstants.ACCOUNT_DISABLED;
+
+/**
+ * Created by samuelgithengi on 1/28/19.
+ */
+public class SyncUtils {
+
+    private static final String TAG = "SyncUtils";
+
+    private org.smartregister.Context opensrpContent = CoreLibrary.getInstance().context();
+
+    private Context context;
+
+    public SyncUtils(Context context) {
+        this.context = context;
+    }
+
+    public boolean verifyAuthorization() {
+        String baseUrl = opensrpContent.configuration().dristhiBaseURL();
+        if (baseUrl.endsWith("/")) {
+            baseUrl = baseUrl.substring(0, baseUrl.lastIndexOf("/"));
```

Comment:
Why not use baseUrl.substring(0, baseUrl.length - 1);?

Currently, it searches the string again for the last occurence of /, right?